### DISCUSSION TITLE
Integration with Station Master Cloud

### DISCUSSION THREAD

**Discussion by timgrossner** [magicbug/Cloudlog#3164] [Ideas / Feature Requests]
Just an idea, Station Master is becoming popular, and an integration with it might not be a bad idea. I know they accept UDP based logging of QSOs, and there is an API as well.

**Comment by magicbug** [comment]
I can't quite figure out what Station Master Cloud is meant to be? also no publicly documented API mentioned anywhere

**Reply by timgrossner** [reply]
yeah, its really sort of the same thing as QRZ but with some extra features. For example. live lists of active stations, with frequency integration from the SM app (if using windows) or from something like I use which is a script that reads my current freq/mode/etc from FLRig and sends it in via UDP. They have a log DB, similar to a QRZ.com, and even have a sync function between SM and QRZ.com. I was using CQRLog to log to SM, and on from there to QRZ, but I want to migrate away from CQRLog to Cloudlog and while the QRZ.com integration is excellent, there is no SM cloud yet, thus my suggestion.
